### PR TITLE
Fix/home registration login page

### PR DESCRIPTION
**Pull request by Zhi0105**
## Summary
- Implementing project logo and landing page background.
- Working with home, registration, login page design and layout.

## Screenshots
home
registration
login

## Reference
- https://tailwindcss.com/docs/installation

### CODE REVIEW

File: app/assets/stylesheets/home.css [hunk 1]
```diff
@@ -1,4 +1,226 @@
-/*
-  Place all the styles related to the matching controller here.
-  They will automatically be included in application.css.
-*/
+/* LOGO ANIMATION START */
+.logo {
+	-webkit-animation: swirl-in-bck 1s ease-out both;
+	animation: swirl-in-bck 1s ease-out both;
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/home.css]: please fix the code alignment

File: app/assets/stylesheets/home.css [hunk 2]
```diff
@@ -1,4 +1,226 @@
...
+    }
+    100% {
+      -webkit-transform: translateY(0);
+              transform: translateY(0);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+  }
+  
+/* SIGN IN ANIMATION END */
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/home.css]: add new line at the end of file

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/home.css (hunk 2 above)
Comment (Zhi0105) [reply app/assets/stylesheets/home.css]: > add new line at the end of file

done

File: app/assets/stylesheets/home.css (hunk 1 above)
Comment (Zhi0105) [reply app/assets/stylesheets/home.css]: done

**Review by analyn-cajocson — APPROVED** [review]

**Review by kitkatpussycat — APPROVED** [review]

**Pull request merged**